### PR TITLE
Add ability to enable or disable analytics collection on Android

### PR DESCRIPTION
**Pull request by MeijerM1**
Implement the enable and disable methods on Android

### CODE REVIEW

**Comment by Mathyn** [comment]
I am encountering the same issue on Android. Can this PR be merged or is something still preventing it? Is there something I can do to help?

**Comment by MeijerM1** [comment]
@stewwan Could you please let us know if you don't have time to do this? Maybe you could consider changing repo ownership to continue development.

File: android/src/main/java/io/stewan/capacitor/analytics/AnalyticsPlugin.java [hunk 1]
```diff
@@ -12,25 +12,26 @@
...
 import com.google.firebase.analytics.FirebaseAnalytics;
+
 import org.json.JSONException;
 import org.json.JSONObject;
+
 import java.util.Iterator;
 
 
 /**
- *
```
Comment (stewones) [inline android/src/main/java/io/stewan/capacitor/analytics/AnalyticsPlugin.java:21]: @MeijerM1 what is the point of this change? why remove my credit?

Comment (stewones) [reply android/src/main/java/io/stewan/capacitor/analytics/AnalyticsPlugin.java:21]: oh sorry I can see you didn't, this review tool on github app is tough

**Review by stewones — APPROVED** [review]

**Comment by stewones** [comment]
published to 2.1.0
thanks @MeijerM1

**Pull request merged**